NineChronicles.Headless/Controllers/GraphQLController.cs

```diff
@@ -127,6 +131,55 @@ public async Task<IActionResult> CheckPeer([FromBody] CheckPeerRequest request)
             }
         }
 
+        [HttpGet(CheckTxId)]
+        public IActionResult CheckTx([FromQuery] string txId)
+        {
+            byte[] txIdByteArray = Enumerable.Range(0, txId.Length)
+                     .Where(x => x % 2 == 0)
+                     .Select(x => Convert.ToByte(txId.Substring(x, 2), 16))
+                     .ToArray();
+            var txIdObject = new TxId(txIdByteArray);
+            if (StandaloneContext.Store is null)
+            {
+                return NotFound("Store is null.");
+            }
+            else
+            {
+                return Ok(StandaloneContext.Store.ContainsTransaction(txIdObject));
+            }
+        }
+
+        [HttpGet(CheckTxId + "/block")]
+        public IActionResult CheckTx([FromQuery] string txId, [FromQuery] string blockHash)
+        {
+            byte[] txIdByteArray = Enumerable.Range(0, txId.Length)
+                     .Where(x => x % 2 == 0)
+                     .Select(x => Convert.ToByte(txId.Substring(x, 2), 16))
+                     .ToArray();
+            var txIdObject = new TxId(txIdByteArray);
+            var blockHashObject = BlockHash.FromString(blockHash);
+            var blockHashList = new List<BlockHash>();
+            blockHashList.Add(blockHashObject);
+
+            if (StandaloneContext.NineChroniclesNodeService is null || StandaloneContext.NineChroniclesNodeService.Swarm is null)
```

Comment:
```suggestion
            if (StandaloneContext.NineChroniclesNodeService is null)
```

[C# `nullable` 기능](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references)를 사용하고 있고 `NineChroniclesNodeService.Swarm` 이 non-nullable이기 때문에 검사하지 않아도 괜찮습니다.

Reply:
답이 늦었습니다 죄송합니다. 아 그럼 조건문을 빼겠습니다 감사합니다!